app/ethtest_helper.go

```diff
@@ -204,31 +203,26 @@ func genesisStateWithValSet(codec codec.Codec, genesisState simapp.GenesisState,
 	genesisState[oracletypes.ModuleName] = codec.MustMarshalJSON(oracleGenesis)
 
 	// operator registration
-	operatorInfos := []operatortypes.OperatorInfo{
+	operatorInfos := []operatortypes.OperatorDetail{
 		{
-			EarningsAddr:     operator.String(),
-			OperatorMetaInfo: "operator1",
-			Commission:       stakingtypes.NewCommission(sdk.ZeroDec(), sdk.ZeroDec(), sdk.ZeroDec()),
+			OperatorAddress: operator.String(),
+			OperatorInfo: operatortypes.OperatorInfo{
+				EarningsAddr:     operator.String(),
+				OperatorMetaInfo: "operator1",
```

Comment:
this this operator1 hardcode

Reply:
This function hasn't been used, it's used for test preparation, so use `opeartor1` for simple meta info.